### DISCUSSION TITLE
Include RWYs with a length of 1,500+ metres in EFB database

### DISCUSSION THREAD

**Discussion by Phil787-9** [magknight/787aviatorsTracker#676] [Ideas]
Could you guys modify the EFB in a way that it incorporates runways with a length of less than 1,900 metres? Took of from PHMU, NTTB, NTTH, and YLHI recently and every time I had to use a different, larger airport in the vicinity as a proxy, because the EFB wouldn't accept the smaller airports' ICAO codes. I think that 1,500 metres+ would be a reasonable threshold

**Comment by aeroniemi** [comment]
same thing as #663
